sycl/test/regression/sycl_api.cpp

```diff
@@ -0,0 +1,176 @@
+// RUN: %clang -std=c++11 -fsycl %s -o %t.out -lstdc++ -lOpenCL
+// RUN: env SYCL_DEVICE_TYPE=HOST %t.out
+// RUN: %CPU_RUN_PLACEHOLDER %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+// RUN: %ACC_RUN_PLACEHOLDER %t.out
+
+//==-- sycl_api.cpp - Regression tests for minor SYCL API bug fixes. -------==//
```

Comment:
Let's specify that this is a regression test for the bug in group API and rename the file accordingly.